buildlib/travis-build

```diff
@@ -20,6 +20,16 @@ cd ../build32
 CC=gcc-6 CFLAGS="-Werror -m32" cmake -GNinja .. -DENABLE_RESOLVE_NEIGH=0
 ninja
 
+# aarch64 build to check compilation on ARM 64bit platform
+cd ../build-aarch64
+if ! (which aarch64-linux-gnu-gcc &> /dev/null); then
+   AARCH64_PATH=`find / -type d -name 'gcc-linaro-*-x86_64_aarch64-linux-gnu' -print -quit 2>/dev/null | xargs echo`
+   export PATH=$PATH:${AARCH64_PATH}/bin
```

Comment:
@jgunthorpe Maybe I'm doing something wrong but as I see, the environment is a bit different.

if I run ```./buildlib/cbuild build-images travis``` it "sudo" and creates the image under root, while the root home is set to "/".

When I run ```./buildlib/cbuild pkg travis``` it runs everything under my current shell user (not root or travis), so the home is quite different.

Travis behavior is a bit different since it runs everything under ~travis so all the files are download to the "right" home. Since during the image creation cbuild I don't know what user is used, I just used default root home "/"